### ISSUE TITLE
Getting devolpment in sync

### ISSUE THREAD

**Issue by Zen4All** [Zen4All-nl/Zen-Cart-Dynamic-Price-Updater#9]
@mc12345678 , @dbltoe, and @torvista

The 4 of us have been active with development of the Dynamic Price Updater module in past years. I have been absent on this one, and mc12345678 has taken over the latest version(s).

I have seen you guys all did your own coding, and updates, but I would like to sync that back in to 1 repo.
Today I made a development branch, to which you can make pr's. The changes I made so far are:
- moving the class code from ```includes/classes/dynamic_price_updater.php``` to ```includes/classes/ajax/zcDPU_Ajax.php```
- Removing the ancient code, using JSON or XML, and replaced it with the build in native Zen Cart Ajax functionality
- Drop support for ZC versions <1.5.6
- PHP version >= 5.6
- Updated optional jquery version to 3.4.1
- Bumped the Module version to 4.0.0 Beta 1

I am working on some proper documentation.

**Comment by torvista** [comment]
Thanks, I'm getting bogged down in pending PR panic, but I'll try.

**Comment by Zen4All** [comment]
> Thanks, I'm getting bogged down in pending PR panic, but I'll try.
:smile: 
You may also point me to the changed code, and I'll merge it for you

**Comment by torvista** [comment]
ok, I can't say how much real changes there are, I fixed what I needed at the time, but didn't go any further. It was too complicated with the forks.

**Comment by Zen4All** [comment]
@torvista thx I'll have a look. I also had a look at MC's fork, but he has so many branches, I have no clue where to start :laughing:

**Comment by torvista** [comment]
> I have no clue where to start
Exactly. That was why I didn't feed back anything. 
I think I used his latest one which fixed the problems I had. Mine still has the specials problem, but I managed to mask it somehow.

**Comment by mc12345678** [comment]
Been updating to the branch that is "latest" when viewing the insights tab.

**Comment by Zen4All** [comment]
@torvista Your updates have been merged in to Beta2
@mc12345678 I'll have a look, and put it in to the next beta

**Comment by mc12345678** [comment]
Note that if making this applicable to ZC 1.5.6 and above that ajax.php in the root of the install should be deleted as that file is maintained by ZC versions and is unnecessary in this package regardless of how one got to this version of the plugin.

**Comment by Zen4All** [comment]
@mc12345678 Your updates have been merged into beta3. How confident atre you that your changes were ready for production?
